synapse/cortex.py

```diff
@@ -5761,10 +5765,25 @@ async def _getMirrorProxy(self):
                 # we are part of the pool and were selected. Convert to local use.
                 return None
 
+            curoffs = await self.getNexsIndx()
+            miroffs = await s_common.wait_for(proxy.getNexsIndx(), timeout)
+            if (delta := curoffs - miroffs) > MAX_NEXUS_DELTA:
+                mesg = (f'Pool mirror [{proxyname}] Nexus offset delta too large '
+                        f'({delta} > {MAX_NEXUS_DELTA}), running query locally.')
+                logger.warning(mesg, extra=await self.getLogExtra(delta=delta, mirror=proxyname, mirror_offset=miroffs))
+                return None
+
+            opts.setdefault('nexsoffs', curoffs - 1)
```

Comment:
Should we setdefault this first and use the return value to calculate the delta since we could potentially have a lower value than the current nexus index?